app/models/manageiq/providers/azure/cloud_manager/refresher.rb

```diff
@@ -2,6 +2,48 @@ module ManageIQ::Providers::Azure
   class CloudManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
     include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
+    def collect_inventory_for_targets(ems, targets)
+      log_header = format_ems_for_logging(ems)
+
+      targets_with_data = targets.collect do |target|
+        target_name = target.try(:name) || target.try(:event_type)
+
+        _log.debug "Collecting inventory for #{log_header}..."
+
+        if refresher_options.try(:[], :inventory_object_refresh)
+          inventory = ManageIQ::Providers::Azure::Builder.build_inventory(ems, target)
+        end
+
+        _log.debug "#{log_header} - Collecting inventory...Complete"
+        [target, inventory]
+      end
+
+      targets_with_data
+    end
```

Comment:
At the moment all it will do is log when inventory collection has started and when it is finished. My thinking was that we could leave this in so that when we do actually support it properly, we won't have to go back and make more changes.